website/docs/getting-started/examples.mdx

```diff
@@ -9,7 +9,7 @@ We also welcome Pull Requests and issues for when they inevitably get neglected
 For more details including a list of examples, refer to the [README].
 
 :::note
-Most of the examples have a live deployment that can be found at https://examples.yew.rs/<example_name>.
```

Comment:
This is trying to say that examples are hosted on `https://examples.yew.rs/<example_name>`.

E.g.: You can visit the router example at: `https://examples.yew.rs/router/`

Reply:
@futursolo I agree with you.
If you click the link `https://examples.yew.rs/` => you get **Page Not Found**
To be more interactive I change it to
`Most of the examples have a live deployment that can be found at [examples](https://github.com/yewstack/yew/tree/yew-v0.20.0/examples).
Click the shield on their individual README page in their respective sub-folder to navigate to the live demo.`

In fact I should correct the link to `https://github.com/yewstack/yew/tree/yew-v0.20.0/examples**#YewExamples**`